torch/autograd/_functions/pointwise.py

```diff
@@ -268,6 +269,30 @@ def backward(ctx, grad_output):
         return grad_output * x.mul(denominator), grad_output * y.neg().mul(denominator)
 
 
+class Erf(Function):
+    @staticmethod
```

Comment:
I think flake8 prefers that there is an empty line just after the class definition, that's what's done in the other classes in this file. Could you add this here and in the `Erfinv` class as well?